### ISSUE TITLE
regex alias addresses don't show up as "me" in the search view

### ISSUE THREAD

**Issue by vrs** [pazz/alot#1230]
When I write emails from regex alias addresses, I see my name instead of "Me" in the search view.
I'm already working on a PR for #1177 that should fix this as well.

**Comment by pazz** [comment]
I agree that a method `Account.matches` or similar, that tests if the account is responsible for a given address is a sensible approach.
